build.gradle

```diff
@@ -7,7 +7,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.6.4'
+        classpath 'com.android.tools.build:gradle:4.1.1'
```

Comment:
try to run `./gradlew clean -Pgroup=com.github.sampingantech -Pversion=1.1.0 install` is it still failing?

Reply:
use previous version `1.1.0` when testing to prevent possibility to upload it (`install` task I'm afraid looks like the equivalent of `build -> then upload to jitpack.io`)

cmiiw

Reply:
will try this @mochadwi

Reply:
I cannot run this script in `Image-Support` project it return error:
```
zsh: permission denied: ./gradlew
```

I also tried to run in `sampingan-android-native` project but it also returned error:
```
FAILURE: Build failed with an exception.

* What went wrong:
Task 'install' is ambiguous in root project 'mobile-android-native'. Candidates are: 'installDebugAndroidTest', 'installGitHook', 'installProdDebug', 'installProdDebugAndroidTest', 'installStagingDebug', 'installStagingDebugAndroidTest'.
```
@mochadwi

Reply:
wait, let's screen.so @anangkur

Reply:
>zsh: permission denied: ./gradlew

well, you might want to add read+write access first, before running the task @anangkur

Reply:
> Task 'install' is ambiguous in root project 'mobile-android-native'. Candidates are: 'installDebugAndroidTest', 'installGitHook', 'installProdDebug', 'installProdDebugAndroidTest', 'installStagingDebug', 'installStagingDebugAndroidTest'.


I think, we'll never able to use `install` task in our Sampingan apps, because it only exist in this `Image-Support` repo

Reply:
This is the reason where `install` task comes: https://github.com/sampingantech/Image-Support/blob/master/support/build.gradle#L2

cmiiw @anangkur

Reply:
Try this instead:

```
./gradlew clean -Pgroup=com.github.sampingantech -Pversion=1.1.0 :support:install
```

is it still failing? @anangkur

Reply:
aahh I see, got it mas @mochadwi I will try it again and add the result here. thanks for the suggestion mas.

Reply:
got this error mas @mochadwi 
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/16663960/104178781-dc463f80-5445-11eb-9c80-f92424322a33.png">

after deleting that class, it run successfully
<img width="1051" alt="image" src="https://user-images.githubusercontent.com/16663960/104178936-16174600-5446-11eb-9049-8acca5ca4e48.png">

Reply:
okaaay awesome, good choice deleting the class :D it's contains no test after all 👍 @anangkur

Reply:
LGTM for me!